exec/util/unit.go

```diff
@@ -43,3 +49,26 @@ func FileFromUnitDropin(root string, unit config.Unit, dropin config.UnitDropIn)
 		},
 	}
 }
+
+func MaskUnit(root string, unit config.Unit) error {
+	filename := filepath.Join(root, SystemdUnitsPath(), string(unit.Name))
+	if err := MkdirForFile(filename); err != nil {
+		return err
+	}
+	return os.Symlink("/dev/null", filename)
+}
+
+func EnableUnit(root string, unit config.Unit) error {
```

Comment:
It *feels* silly for us to be passing root all over the place when it never varies in the course of an ignition invocation.

Perhaps we should have contexts for the packages operating on files needing to construct absolute paths, and all the functions can become methods on the context, with the context creator receiving the root path.

That context could also encapsulate the default directory permissions and make it something changeable @ context create time.